force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +294,10 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
-
-        return null; // Replace null with the variable you used to store the result
+        for (String s : inputList) {
+            stringSet.add(s);
+        }
+        return stringSet; 
```

Comment:
Solid code. (s) Is there a way to do this *without* a loop? Look into how you can instantiate Lists from Sets